app/views/staff/activities/show.html.haml

```diff
@@ -40,14 +41,29 @@
       %h2.govuk-heading-m
         = t("page_content.activity.transactions")
 
-      = link_to(I18n.t("page_content.transactions.button.create"), new_activity_transaction_path(@activity), class: "govuk-button")
-      = render partial: '/staff/shared/transactions/table', locals: { transactions: @transaction_presenters }
+        = link_to(I18n.t("page_content.transactions.button.create"), new_activity_transaction_path(@activity), class: "govuk-button")
```

Comment:
This indentation change looks like it's moved the create link within the header 2 element. I had a look at another of our create buttons and that has it outside of the preceeding header so perhaps this was an accident one?

Reply:
This change was superseded when I refactored the transaction partial